.github/workflows/pr_opex_api_session_manager.yaml

```diff
@@ -0,0 +1,107 @@
+name: PR - Opex Dashboard session-manager
+
+on:
+  pull_request:
+    types:
+      - opened
+      - edited
+      - synchronize
+      - reopened
+    branches:
+      - main
+    paths:
+      - "apps/session-manager/api/*.yaml"
+      - ".github/workflows/pr_opex_api_session_manager.yaml"
+      - "apps/session-manager/.opex/**"
+
```

Comment:
I'd add `./.github/workflows/call_pr_opex_api.yml` too.
Moreover, what should happen if it has changed?